dotcom-rendering/src/web/components/NewsLetterSignupBanner.tsx

```diff
@@ -0,0 +1,76 @@
+import { css } from '@emotion/react';
+import { brand, neutral, space, until } from '@guardian/source-foundations';
+import { SvgGuardianLogo } from '@guardian/source-react-components';
+import { ContainerLayout } from './ContainerLayout';
+import { NewsletterBadge } from './NewslettersBadge';
+
+const mainColWrapperStyle = css`
+	display: flex;
+	height: 100%;
+	flex-direction: column;
+	justify-content: flex-end;
+	max-width: 200px;
+
+	${until.wide} {
+		max-width: 160px;
+	}
+
+	${until.leftCol} {
+		justify-content: flex-start;
+		max-width: 200px;
+	}
+
+	${until.desktop} {
+		max-width: 170px;
+	}
+
+	svg:nth-of-type(1) {
+		width: 65%;
+		display: none;
+
+		${until.leftCol} {
+			display: block;
+		}
+	}
+
+	svg:nth-of-type(2) {
+		padding-top: ${space[1]}px;
+		width: 100%;
+		position: relative;
+		bottom: -11%;
```

Comment:
The values were to align the baseline of the text within the newsletters badge svg with the guardian logo, rather than the bottom of the badge's frame (design requirement). It works as well with `px` so I've changed to that and added a comment about it.